common/tests/interlocked_hl_int/interlocked_hl_int.c

```diff
@@ -192,6 +346,26 @@ TEST_FUNCTION(interlocked_hl_wait_for_not_value_times_out_returns_time_out)
     // cleanup
 }
 
+/*
+Tests:
+InterlockedHL_WaitForNotValue64
+*/
+TEST_FUNCTION(interlocked_hl_wait_for_not_value_64)
+{
+    volatile_atomic int64_t localvalue = 0;
+
+    interlocked_exchange_64(&localvalue, 25);
```

Comment:
>interlocked_exchange_64 [](http://example.com/codeflow?start=4&length=23)

(void) in front, because we are ignoring the return value.

Reply:
Fixed.